src/com/goide/psi/impl/GoPsiImplUtil.java

```diff
@@ -1131,4 +1131,19 @@ public static boolean isUnaryBitAndExpression(@Nullable PsiElement parent) {
     PsiElement grandParent = parent != null ? parent.getParent() : null;
     return grandParent instanceof GoUnaryExpr && ((GoUnaryExpr)grandParent).getBitAnd() != null;
   }
+
+  @Nullable
+  public static PsiElement getBreakStatementOwner(@NotNull GoBreakStatement breakStmt) {
```

Comment:
I guess this method will be needed only for break statement highlighting for a while, so it makes sense to move it to handler